docs/CONFIG-VARS.md

```diff
@@ -239,11 +239,13 @@ V4_CFG_POSTGRES_SERVERS:
 | service_account | External service account for postgres connectivity | string | | false | Required for using cloud-sql-proxy on gcp. See [ansible cloud authentication](user/AnsibleCloudAuthentication.md) | viya |
 | postgres_storage_size | Size of the internal postgreSQL PVCs | string | 128Gi | false |This value can be changed but not decreased after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | backrest_storage_size | Size of the internal pgBackrest PVCs | string | 128Gi | false | This value can be changed but not decreased after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
-| postgres_access_mode | Access mode for the PostgreSQL PVCs | string | ReadWriteOnce | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
-| backrest_access_mode | Access mode for the pgBackrest PVCs | string | ReadWriteOnce | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
+| postgres_access_mode | Access mode for the PostgreSQL PVCs | string | ReadWriteOnce | false | Supported values: [`ReadWriteOnce`,`ReadWriteMany`]. This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
+| backrest_access_mode | Access mode for the pgBackrest PVCs | string | ReadWriteOnce | false | Supported values: [`ReadWriteOnce`,`ReadWriteMany`]. This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | postgres_storage_class | Storage class for the PostgreSQL PVCs | string | sas | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | backrest_storage_class | Storage class for the pgBackrest PVCs | string | sas | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
```

Comment:
Wondering if variables need more descriptive names. For example:

```code
postgres_storage_size -> postgres_pv_storage_size
backrest_storage_size -> backreset_pv_storage_size

etc.
```

This then lets the user know this is an item in the kubernetes cluster that they are setting. I know the description explains this but if I am in the code and see `postgres_access_mode` I am guessing that's something to do with the actual database server and not its storage in the cluster. Where `postgres_pv_access_mode` is a bit clearer.

Reply:
> Having a really hard time with this one. It seems we're now forcing customers to understand their storage class type when creating their postgres servers and these seem like two different questions. @dhoucgitter setup a meeting to discuss. Thx

I'll make an update to the variable name as requested to emphasize the storage aspect and also setting up a discussion with you.

Reply:
@thpang would these name changes for the new postgres vars help emphasize they are PVC and not actual database server items?
I would also make the same name changes for the three vars prefixed with backrest_
```
postgres_storage_size -> postgres_pvc_storage_size
postgres_access_mode -> postgres_pvc_access_mode
postgres_storage_class -> postgres_pvc_storage_class
```

Reply:
yes that's the plan. The information in that block is for the database server. The PVC is only for the internal crunchy database. So any name changes to identify why these items are there would be most helpful.